pytradfri/smart_task.py

```diff
@@ -165,9 +166,14 @@ def set_dimmer_start_time(self, hour, minute):
         """
         #  This is to calculate the difference between local time
         #  and the time in the gateway
-        d1 = self._gateway.get_gateway_info().current_time
-        d2 = dt.utcnow()
-        diff = d1 - d2
+        try:
+            gw = self._gateway.get_gateway_info()
```

Comment:
To get the `GatewayInfo` instance we need to use the api and pass the Command to the api and await this. `Gateway.get_gateway_info` returns a `Command` instance, not `GatewayInfo`.

Reply:
Removed.